lib/active_merchant/billing/gateways/omise.rb

```diff
@@ -0,0 +1,298 @@
+require 'active_merchant/billing/rails'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class OmiseGateway < Gateway
+      API_VERSION = '1.0'
+      API_URL     = 'https://api.omise.co/'
+      VAULT_URL   = 'https://vault.omise.co/'
+
+      STANDARD_ERROR_CODE_MAPPING = {
```

Comment:
atm, this is the only code I could map the code directly to ActiveMerchant's standard error codes.
unless it needed to parse a message.
```
{
  "object": "error",
  "location": "https://docs.omise.co/api/errors#invalid-card",
  "code": "invalid_card",
  "message": "number is invalid and brand not supported (unknown)"
}
```
or
```
{
  "object": "error",
  "location": "https://docs.omise.co/api/errors#invalid-card",
  "code": "invalid_card",
  "message": "expiration date cannot be in the past"
}
```
I'm uncertain about parsing a message, is it good idea to do so? or no need to parse?
@girasquid  what do you think?